frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -644,27 +641,57 @@ function ReaderFooter:updateFooterContainer()
     self[1] = self.footer_positioner
 end
 
-function ReaderFooter:setupAutoRefreshTime()
-    if not self.autoRefreshTime then
-        self.autoRefreshTime = function()
+function ReaderFooter:unscheduleFooterAutoRefresh()
+    if not self.autoRefreshFooter then return end -- not yet set up
+    UIManager:unschedule(self.autoRefreshFooter)
+    logger.dbg("ReaderFooter.autoRefreshFooter unscheduled")
+end
+
+function ReaderFooter:rescheduleFooterAutoRefreshIfNeeded()
+    if not self.autoRefreshFooter then
+        -- Create this function the first time we're called
+        self.autoRefreshFooter = function()
             -- Only actually repaint the footer if nothing's being shown over ReaderUI (#6616)
+            -- (We want to avoid the footer to be painted over a widget covering it - we would
+            -- be fine refreshing it if the widget is not covering it, but this is hard to
+            -- guess from here.)
```

Comment:
FWIW, this wouldn't necessarily be all that great, so all the more reason not to be too bummed about it ;).

As a rather edge-case-y example, the Bookmark code used to triggering footer refreshes on highlight actions: if the footer was invisible, but the highlight in the footer region, it would catch the ReaderUI repaint that updates the highlights, and possibly erase/half-erase/overlay the highlight ;).